pkg/restapi/operation/didexchange/didexchange.go

```diff
@@ -13,6 +13,8 @@ import (
 	"net/http"
 	"time"
 
+	"github.com/hyperledger/aries-framework-go/pkg/wallet"
```

Comment:
should be grouped with the other framework imports.

Reply:
fixed